redisson/src/main/java/org/redisson/connection/SequentialDnsAddressResolverFactory.java

```diff
@@ -91,7 +90,13 @@ public SequentialDnsAddressResolverFactory(int concurrencyLevel) {
 
     @Override
     public AddressResolverGroup<InetSocketAddress> create(Class<? extends DatagramChannel> channelType, DnsServerAddressStreamProvider nameServerProvider) {
-        DnsAddressResolverGroup group = new DnsAddressResolverGroup(channelType, nameServerProvider) {
+        DnsNameResolverBuilder dnsResolverBuilder = new DnsNameResolverBuilder();
```

Comment:
please remove this change since it's already present in project

Reply:
Merged master, this change is gone